modules/clients/src/test/java/org/apache/ignite/jdbc/thin/MvccJdbcTransactionFinishOnDeactivatedClusterSelfTest.java

```diff
@@ -153,7 +154,7 @@ private void deactivateThroughClient() throws Exception {
 
             log.info(">>> Try to deactivate ...");
 
-            state.active(false);
+            state.state(ClusterState.INACTIVE, true);
```

Comment:
Can static import be used? like previuos test used

Reply:
Fixed.